docs/upgrade_1-2_to_1-3.md

```diff
@@ -0,0 +1,79 @@
+# Upgrading from OADP 1.2
+
+> **NOTE:** Always upgrade to next minor version, do NOT skip versions. To update to higher version, do one upgrade at a time. Example: to upgrade from 1.1 to 1.3, upgrade first to 1.2, then to 1.3.
+## Changes from OADP 1.2 to 1.3
+
+- Velero was updated from version 1.11 to 1.12 (Changes reference: https://velero.io/docs/v1.12/upgrade-to-1.12/#upgrade-from-v110-or-higher)
+
+    From this update, OADP now uses Velero Built-in DataMover instead of VSM/Volsync DataMover. This change the following:
+
+    - `spec.features.dataMover` (you can delete secret for `spec.features.dataMover.credentialName` from your cluster, if you want) and `vsm` plugin are not necessary anymore and you need to remove them from your DPA
+
+    - `volsync` operator is not necessary anymore (you can uninstall it from your cluster, if you want)
+
+    - `volumesnapshotbackups.datamover.oadp.openshift.io` and `volumesnapshotrestores.datamover.oadp.openshift.io` CustomResourceDefinitions are not necessary anymore (you can delete `volumesnapshotbackups.datamover.oadp.openshift.io` and `volumesnapshotrestores.datamover.oadp.openshift.io` CRDs from your cluster, if you want)
+
+    Also, OADP now supports a new file system backup software: Kopia.
+
+    - To use it, use the new `spec.configuration.nodeAgent` field. Example
+
+        ```yaml
+        spec:
+          configuration:
+            nodeAgent:
+              enable: true
+              uploaderType: kopia
+        ```
+
+`spec.configuration.restic` field is being deprecated in OADP 1.3, and will be removed in OADP 1.4. To avoid seeing deprecating warnings about it, use the new syntax:
+```diff
+ spec:
+   configuration:
+-    restic:
+-      enable: true
++    nodeAgent:
++      enable: true
++      uploaderType: restic
+```
+
+> **Note:** OADP will be favoring Kopia over Restic in the near future.
+
+## Upgrade steps
```

Comment:
same thing here, lets merge as is. Follow up w/ some doc changes. :) THANK YOU for including it